CUSTOM_BUILDS.md

```diff
@@ -83,6 +83,11 @@ your CloudFormation or Terraform stack to use those new binaries.
 
 ## Using your own binaries in AWS Lambda ##
 
+You can either follow the below process to upload the lambda package to a
+S3 bucket, or compile the binaries as shown [above](#compiling-the-binaries-locally)
+and create a lambda deployment package by zipping (`zip autospotting.zip autospotting`)
```

Comment:
We have the `make archive` target that already creates the zip file, we can just use that.

The file can then be found under the `build` directory.

The `BUCKET_NAME=my-bucket make upload` target will build and then automatically upload it to your own S3 bucket.

Reply:
Ah cool, I didn't know that. Let me add that to the doco.